### DISCUSSION TITLE
Failed to connect with my codespace on CS50

### DISCUSSION THREAD

**Discussion by Soullis** [community/community#43048] [Codespaces]
### Select Topic Area

Bug

### Body

I am not still fully familiar with GitHub and the Codespaces, and today I encountered a problem when entering my codespace with cs50. It was failing to connect, basically, and I did not encounter this error before, any ideas on how to solve it?
Thank you
![Captura de ecrã 2023-01-01 121222](https://user-images.githubusercontent.com/87676378/210175549-4407405c-e502-4f53-858f-dd863c254690.png)

**Comment by kev-out** [comment]
Hello!

Can you let me know what browser you are using while trying to access your codespace? 
I know there are some issues with certain browsers and access to the Codespace
Also are you trying to access this codespace from home or a corporate/school environment?

**Comment by jfullstackdev** [comment]
Codespaces is already fixed, also `github.dev`

**Comment by erikaellr** [comment]
Hi @Soullis, we're thrilled you're using Codespaces, but sorry to hear you were experiencing problems. At the time that you posted, we were actually having an incident that caused failures to connect to codespaces from the web editor. That incident has since been resolved, so I hope that you're now able to connect to any codespaces you're trying to use. If not, please [open a ticket with Support](https://support.github.com/contact).
